### PR TITLE
move create pin link to nav bar and make it permissions based

### PR DESCRIPTION
**Pull request by markmoser**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
       - REACT_APP_AUTH0_DOMAIN: dev-pinster-illuminusltd.auth0.com
       - REACT_APP_AUTH0_AUDIENCE: https://api-dev.pinster.io
       - REACT_APP_AUTH0_SCOPE: openid profile
-      - REACT_APP_AUTH0_RESPONSE_TYPE: token id_token"
```
Comment (scytherswings) [inline .circleci/config.yml]: Why don't you need the id_token? How will we get their name to save as the `display_name`?

Comment (markmoser) [reply .circleci/config.yml]: The auth0 response is including an `idTokenPayload` object even without requesting that. So I sill have `display_name`.

Comment (markmoser) [reply .circleci/config.yml]: rebased to restore this

Comment (scytherswings) [reply .circleci/config.yml]: Oh neat, cool beans

**Pull request merged**